schema/deploy/shipping/views.sql

```diff
@@ -2250,56 +2251,38 @@ create or replace view shipping.scan_hcov19_positives_v1 as
     select
         hcov19_result_release_date,
         count(*) filter (where hcov19_result in ('positive', 'inconclusive')) as total_hcov19_positives,
+        count(*) filter (where hcov19_result = 'negative') as total_hcov19_negatives,
         count(*) filter (where hcov19_result in ('positive', 'inconclusive') and county = 'king') as king_county_positives,
+        count(*) filter (where hcov19_result = 'negative' and county = 'king') as king_county_negatives,
         count(*) filter (where hcov19_result in ('positive', 'inconclusive') and county = 'snohomish') as snohomish_county_positives,
+        count(*) filter (where hcov19_result = 'negative' and county = 'snohomish') as snohomish_county_negatives,
         count(*) filter (where hcov19_result in ('positive', 'inconclusive') and county = 'yakima') as yakima_county_positives,
-        count(*) filter (where hcov19_result in ('positive', 'inconclusive') and county is null) as other_positives
+        count(*) filter (where hcov19_result = 'negative' and county = 'yakima') as yakima_county_negatives,
+        count(*) filter (where hcov19_result in ('positive', 'inconclusive') and county is null) as other_positives,
+        count(*) filter (where hcov19_result = 'negative' and county is null) as other_negatives
     from scan_hcov19_results
     group by hcov19_result_release_date
 ;
 
-comment on view shipping.scan_hcov19_positives_v1 is
+comment on view shipping.scan_hcov19_result_counts_v1 is
   'A view of counts of hcov19 positives from the SCAN project grouped by date results were released.';
```

Comment:
Nipik: Update the comment from positives to results.

Reply:
Oh, will update! Thank you 👍